htdocs/accountancy/class/bookkeeping.class.php

```diff
@@ -2732,23 +2736,38 @@ public function closeFiscalPeriod($fiscal_period_id, $new_fiscal_period_id, $sep
 							$bookkeeping = new BookKeeping($this->db);
 							$bookkeeping->doc_date = $new_fiscal_period->date_start;
 							$bookkeeping->date_lim_reglement = '';
-							$bookkeeping->doc_ref = $new_fiscal_period->label;
+							$bookkeeping->doc_ref = $fiscal_period->label;
```

Comment:
I still have an interrogation with this replacement.
Why $fiscal_period->label instead of $new_fiscal_period->label  ? Done on prupose ?